spec/html/layout-alt-fixed-with-footer-inner-scroll.html

```diff
@@ -25,11 +25,11 @@
     <link rel="apple-touch-icon-precomposed" sizes="57x57" href="../dist/img/apple-touch-icon-precomposed-57.png">
     <link rel="stylesheet" href="../dist/css/patternfly.min.css" >
     <link rel="stylesheet" href="../dist/css/patternfly-additions.min.css" >
-    <script src="../components/jquery/dist/jquery.min.js"></script>
-    <script src="../components/bootstrap/dist/js/bootstrap.min.js"></script>
-    <script src="../components/matchHeight/jquery.matchHeight-min.js"></script>
-    <script src="../components/c3/c3.min.js"></script>
-    <script src="../components/d3/d3.min.js"></script>
+    <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
Are these links going to resolve?  Looks like a lot of them changed.  If they do resolve, getting rid of the double slash would make it less confusing to the reader.